tgt_grease/enterprise/Model/DeDuplication.py

```diff
@@ -361,8 +359,47 @@ def object_field_score(collection, ioc, source_name, configuration_name, obj, ob
                 continue
         if len(field_scores) is 0:
             return 0.0
-        else:
-            return float(sum(field_scores) / float(len(field_scores)))
+        return float(sum(field_scores) / float(len(field_scores)))
+
+    @staticmethod
+    def make_hashable(obj):
+        """Takes a dictionary and makes a sorted tuple of strings representing flattened key value pairs
+        Args:
+            obj (dict): A dictionary
+        Returns:
+            tuple<str>: a sorted flattened tuple of the dictionary's key value pairs
+
+        Example:
+            {
+                "a": ["test1", "test2"],
+                "b": [{"test2": 21}, {"test1": 1}, {"test7": 3}],
+                "c": "test"
+            }
+            becomes...
+            (('a', ('test1', 'test2')),
+             ('b', ((('test1', 1),), (('test2', 21),), (('test7', 3),))),
+             ('c', 'test'))
+        """
+        final = []
+        sorted_tuples = Deduplication.make_hashable_helper(obj)
+        print(sorted_tuples)
```

Comment:
Please remove prints and instead use log statements with appropriate levels and verbose/trace flags